bin/navuser

```diff
@@ -77,6 +77,19 @@ def adduser(args):
     print(msg % args.login, file=sys.stderr)
 
 
+def removeuser(args):
+    try:
+        args.login.delete()
+    except args.login.DoesNotExist:
+        print(
+            "User %s is not registered, cannot remove" % args.login.login,
+            file=sys.stderr,
+        )
+        sys.exit(1)
+    else:
+        print("User %s has been removed" % args.login.login, file=sys.stderr)
```

Comment:
```suggestion
    args.login.delete()
    print("User %s has been removed" % args.login.login, file=sys.stderr)
```

We don't need this error handling since `usergetter` does already handle if a user doesn't exist